client/blocks/login/style.scss

```diff
@@ -2,7 +2,7 @@
 .login {
 	&.is-jetpack {
 		.button.is-primary {
-			background-color: $green-jetpack;
+			background-color: var( --color-jetpack );
 			border-color: darken( $green-jetpack, 5% );
```

Comment:
@drw158 do you have feedback on how to address those? I'm not sure how I feel about lighten or darken brand colors but if we do so I guess we should pre-define them as the others, like `--color-jetpack-light` or `--color-jetpack-dark`.

Reply:
There's three different ones in total, btw:

darken( $green-jetpack, 30% );
darken( $green-jetpack, 5% );
lighten( $green-jetpack, 25% );

Reply:
@jeffgolenski We are replacing all scss variables with css custom props. I'm wondering if Jetpack has official shades of the Jetpack Green, or is it ok to generate a couple of shades for the purposes of the buttons in Calypso?

Reply:
@drw158 We only have one "official" jetpack green now. Feel free to explore some different shades as you see fit. the light and darken colors mentioned above aren't really official, but we use them in a couple places.

Reply:
If this is confined to just the button styles, I think defining dark and light colors for Jetpack is fine (-color-jetpack-light or --color-jetpack-dark). Jetpack is a unique case because its color is not from the Color Studio palette and thus doesn't have 10 shades like the other colors. Jetpack green is also unique because it needs more shades to create button styles.

Let's use `#81d677` for light and `#2e8928` for dark